src/index.html

```diff
@@ -10,23 +10,36 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <main class="main">
+      <form class="search-bar">
+        <label for="id1"></label>
+        <input
+          id="id1"
+          type="text"
+          data-qa="big"
+          action="#"
```

Comment:
HTML Content: The `action` attribute is not applicable to the `input` element. It should be used within the `form` tag to specify the URL of the server-side script that will process the submitted form data.